go/consensus/api/api_test.go

```diff
@@ -0,0 +1,37 @@
+package api
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestSlashReason(t *testing.T) {
```

Comment:
Probably should not be called TestSlashReason (also there's some "reason" references below) :-)